lib/chef/provider/file.rb

```diff
@@ -230,7 +230,7 @@ def inspect_existing_fs_entry
         elsif file_class.symlink?(new_resource.path) && new_resource.manage_symlink_source.nil?
           logger.warn("File #{path} managed by #{new_resource} is really a symlink (to #{file_class.realpath(new_resource.path)}). Managing the source file instead.")
           logger.warn("Disable this warning by setting `manage_symlink_source true` on the resource")
-          logger.warn("In a future Chef release, 'manage_symlink_source' will not be enabled by default")
+          logger.warn("In a future #{Chef::Dist::PRODUCT} release, 'manage_symlink_source' will not be enabled by default")
```

Comment:
This reference could probably be removed.

```suggestion
          logger.warn("In a future release, 'manage_symlink_source' will not be enabled by default")
```